Makefile

```diff
@@ -27,6 +27,9 @@ bootstrap:
 test: clean bootstrap
 	$(BUILD_TOOL) $(XCODEFLAGS) test
 
+coverage:
+	swiftcov generate --output coverage --threshold 0 $(BUILD_TOOL) $(XCODEFLAGS) test -- ./Source/SwiftCovFramework/BuildSettings.swift ./Source/SwiftCovFramework/CoverageReporter.swift ./Source/SwiftCovFramework/Shell.swift ./Source/SwiftCovFramework/SimCtl.swift ./Source/SwiftCovFramework/Xcodebuild.swift
```

Comment:
It did not work well at first I wrote so, but it works now. Thanks for finding out. I'll fix it.